types/fluent-ffmpeg/tsconfig.json

```diff
@@ -1,22 +1,22 @@
 {
-    "compilerOptions": {
```

Comment:
that just might be my editor doing an auto format. Doesn't look like anything really changed other than spacing.